### DISCUSSION TITLE
Shelly 2.5 e pulsante ZigBee

### DISCUSSION THREAD

**Discussion by lucaxxaa** [indomus/forum#102] [Hardware & Firmware]
Volevo installare shelly 2.5 per tapparella elettrica. 
1- per integrare lo shelly 2.5 userei integrazione nativa.  Devo fare altre configurazioni in home assistat, per poi usare la tapparella o avrei gia tutti i sensori pronti da usare ( in rete ho trovato spiegazioni su mqtt, che non uso ) 

2-e' possibile per comandare lo shelly 2.5  e quindi la tapparella, installando unicamente un interruttore Zigbee? 
Sarà possibile in questo caso alzare la tapparella a metà e poi bloccarla?
Grazie

**Comment by snakuzzo** [comment]
1. Se usi l'integrazione nativa Shelly di HA non devi fare niente...ti apparirà direttamente l'entità di tipo cover

2. Se vuoi device Zigbee puoi usare questo... https://www.zigbee2mqtt.io/devices/14594.html

**Reply by lucaxxaa** [reply]
Bello. Questo Vimar andrà anche con zha ?

Stavo leggendo che si possono usare anche i tradifari Ikea, quelli usati per fare il dimmer delle lampade

**Reply by snakuzzo** [reply]
Non saprei... per ZHA è da provare...per quanto riguarda la roba di IKEA non saprej proprio.
Dai anche un'occhiata [QUI](https://zigbee.blakadder.com/covers.html)   comunque

**Comment by lucaxxaa** [comment]
Sui switch Ikea tradifari leggevo che funzionerebbero anche bene cioè avendo due pulsanti si può tenere premuto su uno per alzare la tapparella è a rilascio si ferma, ma hanno un problema che hanno la batteria che si scarica molto in fretta

**Reply by snakuzzo** [reply]
Non ho capito il tuo discorso. Vorresti accompagnare gli shelly con dei pulsanti ikea zigbee?

**Reply by lucaxxaa** [reply]
guarda questo blueprint .
In pratica comandi la tapparella elertricca con il pulsante remoto zigbee

https://community.home-assistant.io/t/zha-deconz-zigbee2mqtt-ikea-e1743-on-off-switch-dimmer-universal-blueprint-all-actions-double-click-events-control-lights-media-players-and-more-with-hooks/277861

**Reply by snakuzzo** [reply]
Sì ma permettimi di dire che quella è una banalità. Ho fatto la stessa cosa con lo stesso pulsante per dimmerare e pilotare dei led tramite un'automazione.
Resta il fatto che devi comunque applicare un attuatore per pilotare le tapparelle (es.Shelly/Sonoff/altri device come quello che ti ho mostrato della vimar).

**Reply by lucaxxaa** [reply]
...ceerto. Stavo cercando di capire se, installando solamente shelly 2.5 nel cassonetto collegato al motore della tapparella, e per evitare di scappare il muro e mettere una scatola 503 e, usare solamente quel pulsante zigbee in wireless abbinato ad HA e shelly 2.5 ( anche se ovviamente se non andasse esempio HA, non potrei usare le tapparelle)

**Reply by snakuzzo** [reply]
se non va HA (o qualsiasi sia il punto di contatto tra i device) t'attacchi.
Resta il fatto che, mal che vada, puoi collegarti alla webgui dello shelly e dare i comandi.

**Reply by lucaxxaa** [reply]
...e sii, infatti mi sa che spacchero e mettero le scatole 503 ....

Comandarli da webgui intendi da web e da app shelly, giusto...e in ogni caso, esempio router che non funziona, tapparella che non va. Quindi meglio mettere il pulsante fisico

**Reply by snakuzzo** [reply]
sì...meglio.
